TerminalDocs/json-fragment-extensions.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Windows Terminal JSON Fragment Extensions
+description: Learn how to use JSON fragment extensions in Windows Terminal.
+author: cinnamon-msft
+ms.author: cinnamon
+ms.date: 02/25/2021
+ms.topic: how-to
+---
+
+# JSON fragment extensions in Windows Terminal
```

Comment:
We should add an FAQ section here or in the main FAQ page specifically for fragment extensions. I have the following questions:
- user
  - Why am I getting more profiles with the "source" property?
  - Why is my \<insert random setting\> not being set by the extension? (answer: check profiles.defaults)
- extension author:
  - Can I control what the "source" property is?
  - Can I provide assets in my extension? Like an icon or background image?
  - What happens if I reference an invalid color scheme? Will WT crash?
Just a few corner cases that I feel we should mention somewhere.